README.md

```diff
@@ -2,56 +2,68 @@
-application using Django.  It is not intended to be visible to the
-general public, or even to instructors (at least initially — we may add that
-later).  Instead, the target audience is administrators, most of whon are
-non-programmers, who need to keep track of what workshops are being arranged,
-when they're supposed to occur, who's teaching what, and so on.
+Amy is a web-based workshop administration application for [Software
+Carpentry][swc] and related projects.  Its target audience is workshop
+coordinators, most of whon are non-programmers, who need to keep track
+of what workshops are being arranged, when they're supposed to occur,
+who's teaching what, and so on.
 
-# Getting started
+Amy is built using [Django][django], with a bit of Javascript and
+other things thrown in.  If you would like to help, please read:
 
-1. Clone the repository:
+*   the setup instructions below,
+
+*   the [contributor guidelines](CONTRIBUTING.md), and
+
+*   our [contributor code of conduct](CONDUCT.md).
+
+Please [check with us][contact-address] or open an [issue][issues]
+before starting work on new features.
+
+## Getting Started
+
+1.  Clone the repository:
 
     ~~~
     $ git clone https://github.com/swcarpentry/amy.git
     $ cd amy
     ~~~
 
-2. Install Django and other dependencies:
+2.  Install Django and other dependencies:
 
     ~~~
     $ python3 -m pip install --user -r requirements.txt
     ~~~
 
-    If you're experienced Python programmer, feel free to create a Python3-compatible [virtualenv](https://virtualenv.pypa.io/en/latest/userguide.html) for Amy and install dependencies from `requirements.txt`.
+    If you're experienced Python programmer, feel free to create a
+    Python3-compatible [virtualenv][virtualenv] for Amy and install
+    dependencies from `requirements.txt`.
 
-3. Install [Bower](http://bower.io/) — the tool that manages Amy's JavaScript and CSS dependencies:
+3.  Install [Bower][bower], the tool that manages Amy's JavaScript and CSS dependencies:
 
     ~~~
     $ sudo npm install -g bower
     ~~~
 
-    You may need some additional dependencies to install [Bower](http://bower.io/), such as [NodeJS](https://nodejs.org/) and [npm](https://www.npmjs.com/).
+    You may need some additional dependencies to install [Bower][bower], such as [NodeJS][nodejs] and [npm][npm].
 
-    **Note**: if you don't want to use `sudo`, you can install `bower` locally. You'll need to set up your `$PATH` correctly, though. Look [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npm-s-default-directory-to-another-directory) for details.
+    **Note**: if you don't want to use `sudo`, you can install `bower`
+    locally. You'll need to set up your `$PATH` correctly, though. Look
+    [here][fixing-npm-permissions] for details.
 
-4. Setup your local database with redacted (development-ready) data with:
+4.  Set up your local database wit redacted (development-ready) data with:
```

Comment:
Still seeing "wit" here. I'm guessing you did some sort of a merge instead of rebase.